docs/game-jam.md

```diff
@@ -1,130 +1,137 @@
 * cardType: sharedExample
-* imageUrl: https://pxt.azureedge.net/api/56203-12068-28996-67940/thumb
+* imageUrl: https://pxt.azureedge.net/api/70057-29128-20368-99273/thumb
 ---
-* name: Timing and Patience
-* description: A simple, addicitive game where you must use timing and patience to try and nab the high score. Try out the different difficulty levels!
-* author: EgeoTube Gamez
-* url: https://arcade.makecode.com/28170-65925-97085-38467
+* name: S&A - Prechistoric World
+* description: Find the missing battery for your time machine in this sidescrolling adventure game! Created by Lisiu23
+* author: Lisiu23
+* url: https://arcade.makecode.com/57672-87358-35699-32284
 * cardType: sharedExample
-* imageUrl: https://pxt.azureedge.net/api/28170-65925-97085-38467/thumb
+* imageUrl: https://pxt.azureedge.net/api/57672-87358-35699-32284/thumb
 ---
-* name: Time Over
-* description: A beautiful game with a great premise! Collect all the numbers for your clock before time runs out!
-* author: Daniela
-* url: https://arcade.makecode.com/09641-64692-55182-88904
+* name: Dinosaur Hunting
+* description: Fight dinosaurs and ghosts in this classic SHMUP game! Created by Sayem
+* author: Sayem
+* url: https://arcade.makecode.com/22703-75962-83076-91368
 * cardType: sharedExample
-* imageUrl: https://pxt.azureedge.net/api/09641-64692-55182-88904/thumb
+* imageUrl: https://pxt.azureedge.net/api/22703-75962-83076-91368/thumb
 ---
-* name: fast restaurant
-* description: Deliver the right dishes to the customer as soon as possible in this creative, challenging game.
-* author: demoCrash
-* url: https://arcade.makecode.com/61260-89058-32987-83888
+* name: Dino's Great adventure
+* description: Play as a mother dino finding a place to lay her egg just after the meteor struck the earth 65 million year ago in this platformer game! Created by Bifrost22
+* author: Bifrost22
+* url: https://arcade.makecode.com/69680-60757-30003-05179
 * cardType: sharedExample
-* imageUrl: https://pxt.azureedge.net/api/61260-89058-32987-83888/thumb
+* imageUrl: https://pxt.azureedge.net/api/69680-60757-30003-05179/thumb
 ---
-* name: 4-in-1 time-related 8-bit fun
-* description: Four fun, unique games for the price of one! Make sure you play through each of these time-related minigames.
-* author: Samuel G
-* url: https://arcade.makecode.com/19854-09262-25388-69397
+* name: Prehistoric Explorer
+* description: Travel back in time and dodge dinosaurs while trying to get home in this adventure game! Created by catstudio Games
+* author: catstudio Games
+* url: https://arcade.makecode.com/52290-68197-81450-36464
 * cardType: sharedExample
-* imageUrl: https://pxt.azureedge.net/api/19854-09262-25388-69397/thumb
+* imageUrl: https://pxt.azureedge.net/api/52290-68197-81450-36464/thumb
 ---
 * name: See more...
-* description: Check out all the entries on the official game jam page!
+* description: Check out all the entries on the official game jam page! Created by sylvancircle
```

Comment:
Accidentally left in "Created by..." although I don't think this description actually shows up anywhere.